ci/docker/arch-linux

```diff
@@ -0,0 +1,22 @@
+FROM public.ecr.aws/docker/library/archlinux:latest 
+
+RUN pacman -Sy --noconfirm git 
+RUN git clone --recurse-submodules https://github.com/aws/aws-sdk-cpp.git
+RUN pacman -Sy --noconfirm \
+    cmake \
+    ninja \
+    clang \
+    curl \
+    zip
+    
+
+# Note: (2022-08-23)
+# Using -DUSE_OPENSSL=OFF as a workaround to an AWS SDK dependency issue with this distro.
+# The current SDK version has a dependency on a static build version of openssl, not available through pacman.
```

Comment:
It's not available as a built package from the main repositories. If you build [openssl-static](https://aur.archlinux.org/packages/openssl-static) from the AUR with [`makepkg`](https://wiki.archlinux.org/title/Makepkg#Usage), the resulting package can be installed with `pacman -U <package-file>`.